src/Lib/Canvas.vala

```diff
@@ -153,70 +153,88 @@ public class Akira.Lib.Canvas : Goo.Canvas {
             motion_hover_event (event);
             return false;
         }
+        var canvas_event_x = event.x;
+        var canvas_event_y = event.y;
+        var event_x = canvas_event_x;
+        var event_y = canvas_event_y;
```

Comment:
this block of clode was a bit weird. it may be easier to understand if you all setts come from the same variable like:

```
        var canvas_event_x = event.x;
        var canvas_event_y = event.y;
        var event_x = event.x;
        var event_y = event.y;
```

Reply:
it tries to reflect `event_y` comes from `canvas_event_x` to be in sync with rest of variable naming.

I tried to get rid of canvas_space or item_space variables, but something don't work as expected. Transforms affect object so item_space is transformed, sometimes is useful (translate) and others is not (rotate)

Let's fix the other obvious things and will come back to this later. I'd want to get rid of it too.